test_integration/geopm_test_launcher.py

```diff
@@ -199,7 +199,7 @@ def check_run(self, test_name):
                                                          self._node_list, self._host_file)
             launcher.run(stdout=outfile, stderr=outfile)
 
-    def run(self, test_name, include_geopm_policy=True):
+    def run(self, test_name, include_geopm_policy=True, add_geopm_args=[]):
```

Comment:
the back door is open!

Reply:
I would really like to fix up the test launcher.  There are multiple different ways of handling the geopm command line args.